### PR TITLE
Fix consents not updating after warning/confirmation message improvements

### PR DESCRIPTION
**Pull request by ayshsandu**
To update consents from user dashboard

### CODE REVIEW

File: modules/jaggery-apps/portal/gadgets/consent_management/js/gadget.js [hunk 1]
```diff
@@ -385,18 +386,18 @@ function addActions(container) {
     });
 }
 
-function showWarning() {
+function showWarning(container) {
```
Comment (pulasthi7) [inline modules/jaggery-apps/portal/gadgets/consent_management/js/gadget.js]: what's the usage of container here?

Comment (ayshsandu) [reply modules/jaggery-apps/portal/gadgets/consent_management/js/gadget.js]: container is an element need to be passed in the callBack function of the warning/confirmation dialog

Comment (pulasthi7) [reply modules/jaggery-apps/portal/gadgets/consent_management/js/gadget.js]: But the callback function is empty here. It's used in showConfirm()

Comment (ayshsandu) [reply modules/jaggery-apps/portal/gadgets/consent_management/js/gadget.js]: Fixed

**Pull request merged**